scripts/ci.js

```diff
@@ -0,0 +1,41 @@
+/* globals process */
+/* eslint-disable no-console */
+
+const childProcess = require('child_process');
+
+/**
+ * Executes the passed in command-line command and logs its output to standard
+ * out.
+ *
+ * @param {string} command
+ * @returns {void}
+ */
+function run(command) {
+    console.log('Executing command', command);
+
+    childProcess.execSync(command,
+        { stdio: [ process.stdin, process.stdout, process.stderr ] });
+
+    console.log('Completed command', command);
+}
+
+run('npm install');
+run('npm run lint');
+run('npm test');
+run('npm run build-prod');
+
+console.log('Starting test server');
+
+const testServer = childProcess.spawn('python', [
```

Comment:
it looks like this would be handy to have as a target in package.json, and it could take the port as an argument